basic/model.py

```diff
@@ -209,7 +209,7 @@ def _build_forward(self):
         passage_len = tf.reduce_sum(tf.cast(self.passage_mask, 'int32'), 2)
         # [batch_size]
         question_len = tf.reduce_sum(tf.cast(self.question_mask, 'int32'), 1)
-        # [batch_size, num_sentences]
+        # [batch_size, num_options]
```

Comment:
note that `options_mask` is of shape `[batch_size, num_options, option_size]`. This sums it up so we have a matrix of `[batch_size, num_options]` indicating the length of each option. I hackily use this as a mask for the option logits later down the line (by converting the length matrix of to a bool matrix representing whether we have an option there, or whether it's all padding)